scripts/etc/init.d/rcS

```diff
@@ -838,29 +838,43 @@ echo "OK"
 # networking
 echo -n "  Configuring network settings... "
 touch /rootfs/etc/network/interfaces || fail
-{
-    # wheezy seems to add the lo interface, so first check for it
-    if ! grep -q "auto lo" /rootfs/etc/network/interfaces; then
-        echo "auto lo"
-        echo "iface lo inet loopback"
-        echo ""
-    fi
+# lo interface may already be there, so first check for it
+if ! grep -q "auto lo" /rootfs/etc/network/interfaces; then
+    echo "auto lo"                      >> /rootfs/etc/network/interfaces
+    echo "iface lo inet loopback"       >> /rootfs/etc/network/interfaces
+    echo ""                             >> /rootfs/etc/network/interfaces
+fi
 
-    # eth0 config
-    echo "auto $ifname"
-    if [ "$ip_addr" = "dhcp" ]; then
-        echo "iface $ifname inet dhcp"
+# eth0 config
+echo "auto $ifname"                     >> /rootfs/etc/network/interfaces
+if [ "$ip_addr" = "dhcp" ]; then
+    echo "iface $ifname inet dhcp"      >> /rootfs/etc/network/interfaces
+else
+    echo "iface $ifname inet static"    >> /rootfs/etc/network/interfaces
+    echo "    address $ip_addr"         >> /rootfs/etc/network/interfaces
+    echo "    netmask $ip_netmask"      >> /rootfs/etc/network/interfaces
+    echo "    broadcast $ip_broadcast"  >> /rootfs/etc/network/interfaces
+    echo "    gateway $ip_gateway"      >> /rootfs/etc/network/interfaces
+fi
+
+# wlan config
+if [ "$ifname" != "eth0" ]; then
+    # Install wireless dependencies
+    echo -n "Installing wpasupplicant package... "
+    chroot /rootfs /usr/bin/apt-get -y install wpasupplicant >& /dev/null
+    if [ $? -eq 0 ]; then
+        echo "OK"
     else
-        echo "iface $ifname inet static"
-        echo "    address $ip_addr"
-        echo "    netmask $ip_netmask"
-        echo "    broadcast $ip_broadcast"
-        echo "    gateway $ip_gateway"
+        echo "FAILED !"
     fi
-    if [ "$ifname" != "eth0" ]; then
-        echo "wpa-conf /boot/config/wpa_supplicant.conf"
+    # 
+    if [ -e /bootfs/config/wpa_supplicant.conf ]; then
+        # copy the installer version of `wpa_supplicant.conf`
+        cp /bootfs/config/wpa_supplicant.conf /rootfs/etc/wpa_supplicant/
     fi
-} >> /rootfs/etc/network/interfaces || fail
+    echo "wpa-conf /etc/wpa_supplicant/wpa_supplicant.conf"	>> /rootfs/etc/network/interfaces
```

Comment:
remove this line, and after running `post-install.txt`, check for the existence of the `/rootfs/etc/wpa_supplicant/wpa_supplicant.conf` file (it might be put there using `install_files`, even if not used in the installer as such)... if found, add the appropriate entry to `/rootfs/etc/network/interfaces`